### ISSUE TITLE
[REQ] Improvements to the option to inspect storage units

### ISSUE THREAD

**Issue by crashzk** [ianlucas/cs2-inventory-simulator#33]
Currently on the website, when we click to inspect a storage unit, it doesn't matter if it has items inside it or not, it opens normally.

In CS2, when it is empty, it shows a message about how the storage unit works.

![image](https://github.com/ianlucas/cs2-inventory-simulator/assets/32937653/bb15d922-9463-4e30-bdc3-fc81e8ee69af)

Would this feature be possible on the website? The only change would be in the number of items shown, as we can change how many items will be able to be placed in the storage unit through the database with the `InventoryStorageUnitMaxItems` option, it would be interesting to show the number that we configure in it.

In other words, instead of showing 1,000 items, it would show 32 items or whatever number we configure.
